packages/core/src/app/shipping/StripeStateMapper.ts

```diff
@@ -0,0 +1,232 @@
+export default function StripeStateMapper(country: string, state: string): string {
```

Comment:
what's the purpose of this file? Can't we use the geographies data?

Reply:
Hey @animesh1987, the purpose of this file is to map the state that Stripe is sending us because it returns a non-standard format example: For the country MX, if the user wants to select "Ciudad de Mexico" Stripe returns "CDMX" as the state code, on the other hand, BC uses "MEX" as state code